examples/direct-flp/prometheus-local/run-example.sh

```
@@ -0,0 +1,43 @@
+AGENT_BIN="${REPO_ROOT}/bin/netobserv-ebpf-agent"
+
+if [[ ! -x "${AGENT_BIN}" ]]; then
+  echo "Missing ${AGENT_BIN}. Run: cd ${REPO_ROOT} && make compile" >&2
+  exit 1
+fi
+
+cd "${SCRIPT_DIR}"
+docker compose up -d
+
+export EXPORT=direct-flp
+export FLP_CONFIG="$(cat "${SCRIPT_DIR}/flp-prometheus.yaml")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)